packages/sources/svelte/src/components/structure/VtmnListItem/VtmnListItem.svelte

```diff
@@ -63,25 +83,57 @@
   aria-disabled={disabled}
   {...$$restProps}
 >
-  {#if checkSlotExists('start-visual')}
-    <div class="vtmn-list_start-visual">
-      <slot name="start-visual" />
-    </div>
-  {/if}
+  {#if href}
+    <a
+      class="vtmn-list__link"
+      {href}
+      {target}
+      rel={computeRel(target, rel)}
+      aria-disabled={disabled}
+    >
+      {#if checkSlotExists('start-visual')}
+        <div class="vtmn-list_start-visual">
+          <slot name="start-visual" />
+        </div>
+      {/if}
 
-  {#if checkSlotExists('text')}
-    <div class="vtmn-list_text">
-      <slot name="text" />
+      {#if checkSlotExists('text')}
+        <div class="vtmn-list_text">
+          <slot name="text" />
 
-      {#if checkSlotExists('subtext')}
-        <slot name="subtext" />
+          {#if checkSlotExists('subtext')}
+            <slot name="subtext" />
+          {/if}
+        </div>
       {/if}
-    </div>
-  {/if}
 
-  {#if checkSlotExists('end-action')}
-    <div class="vtmn-list_end-action">
-      <slot name="end-action" />
-    </div>
+      {#if checkSlotExists('end-action')}
+        <div class="vtmn-list_end-action">
+          <slot name="end-action" />
+        </div>
+      {/if}
+    </a>
+  {:else}
+    {#if checkSlotExists('start-visual')}
+      <div class="vtmn-list_start-visual">
+        <slot name="start-visual" />
+      </div>
+    {/if}
+
+    {#if checkSlotExists('text')}
+      <div class="vtmn-list_text">
+        <slot name="text" />
+
+        {#if checkSlotExists('subtext')}
+          <slot name="subtext" />
+        {/if}
+      </div>
+    {/if}
+
+    {#if checkSlotExists('end-action')}
+      <div class="vtmn-list_end-action">
+        <slot name="end-action" />
+      </div>
+    {/if}
```

Comment:
Not possible with svelte. 
The only way to avoid this is create another component with 

```
  {#if checkSlotExists('start-visual')}
    <div class="vtmn-list_start-visual">
      <slot name="start-visual" />
    </div>
  {/if}

  {#if checkSlotExists('text')}
    <div class="vtmn-list_text">
      <slot name="text" />

      {#if checkSlotExists('subtext')}
        <slot name="subtext" />
      {/if}
    </div>
  {/if}

  {#if checkSlotExists('end-action')}
    <div class="vtmn-list_end-action">
      <slot name="end-action" />
    </div>
  {/if}
```

And inject it link 

```
<li
  on:click
  class={componentClass}
  role="option"
  tabindex="0"
  aria-disabled={disabled}
  {...$$restProps}
>
  {#if href}
    <a
      class="vtmn-list__link"
      {href}
      {target}
      rel={computeRel(target, rel)}
      aria-disabled={disabled}
    >
    <ListContent>
      <slot name="start-visual" />
      <slot name="text" />
      <slot name="subtext" />
      <slot name="end-action" />
    </ListContent>
  </a>
{:else}
    <ListContent>
      <slot name="start-visual" />
      <slot name="text" />
      <slot name="subtext" />
      <slot name="end-action" />
    </ListContent>
{/else}
</li>
```